pages/api/play-attempt/index.ts

```diff
@@ -263,19 +278,14 @@ export default withAuth({
 
         // if it has been more than 15 minutes OR if we have no play attempt record create a new play attempt
         // increment the level's calc_playattempts_count
-        let incr = {};
-
         if (!statRecord?.complete) {
-          incr = { $inc: {
-            calc_playattempts_count: 1,
-          } };
+          await LevelModel.findByIdAndUpdate(levelId, {
+            $inc: {
+              calc_playattempts_count: 1,
```

Comment:
can we delete this? i don't see calc_playattempts_count used anywhere in the codebase besides being incremented